src/VPolygon.jl

```diff
@@ -14,10 +16,10 @@ Type that represents a polygon by its vertices.
 
 ### Notes
 
-The constructor of `VPolygon` runs a convex hull algorithm, and the given
-vertices are sorted in counter-clockwise fashion.
-The constructor flag `apply_convex_hull` can be used to skip the computation of
-the convex hull.
+The constructor of `VPolygon` runs a convex hull algorithm on its vertices by
+default, to remove the possibly redundant constraints. The vertices are sorted in
```

Comment:
> to remove the possibly redundant constraints

This is confusing for a `VPolygon`. It sounds as if we compute the constraints.

Reply:
changed to: "The constructor of `VPolygon` runs a convex hull algorithm on its vertices by
default, to remove the possibly redundant vertices."

Reply:
Nothing has changed yet :wink:

Reply:
could you check again now?